bundles/org.openhab.binding.netatmo/README.md

```diff
@@ -36,49 +23,73 @@ The variables you will need to get to setup the binding are:
 * `<USERNAME>` The username you use to connect to the Netatmo API (usually your mail address).
 * `<PASSWORD>` The password attached to the above username.
 
+The binding has the following configuration options:
 
-### 2. Bridge and Things Configuration
+| Parameter    | Type          | Description                                                                                |
+|--------------|---------------|--------------------------------------------------------------------------------------------|
+| features     | String        | The perimeter of functionalities given to the binding WEATHER, AIR_CARE, ENERGY, SECURITY  |
+| readFriends  | Boolean       | Enables or disables the discovery of guest weather stations.                               |
 
-Once you will get needed informations from the Netatmo API, you will be able to configure bridge and things.
 
-E.g.
+## Bridge Configuration
+
+You will have to create at first a bridge to handle communication with your Netatmo Application.
+
+The Account bridge has the following configuration options:
+
+-   **clientId:** Client ID provided for the application you created on http://dev.netatmo.com/createapp.
+-   **clientSecret:**  Client Secret provided for the application you created.
+-   **username:** Your Netatmo API username (email).
+-   **password:** Your Netatmo API password.
+-   **webHookUrl:** Protocol, public IP and port to access openHAB server from Internet.
+-   **reconnectInterval:** The reconnection interval to Netatmo API (in s).
+
+
+## List of supported things
+
+| Parameter       | Type   | Netatmo Object | Description                                                                                              |
```

Comment:
Parameter is a bad title, I would rather use "thing type".

And can you at least add another column in the table to list the different thing parameters, maybe just the parameters ids.

Reply:
Updated via your PR.

Reply:
The list of parameter ids is still missing.
I just updated the column title.

Reply:
PR submitted.